docs/markdown/Reference-manual.md

```diff
@@ -655,10 +656,10 @@ be passed to [shared and static libraries](#library).
   target by overriding the default. By default on Windows this is
   `exe` and on other platforms it is omitted. Set this to `[]`, or omit
   the keyword argument for the default behaviour.
-- `override_options` *(since 0.40.0)*: takes an array of strings in the same format as
-  `project`'s `default_options` overriding the values of these options
-  for this target only.
-- `gnu_symbol_visibility` *(since 0.48.0)*: specifies how symbols should be exported, see
+- `override_options` takes an array of strings or dictionary in the same
+  format as `project`'s `default_options` overriding the values of these
+  options for this target only, since 0.40.0. Dictionary new in 0.56.0
```

Comment:
I find the "since" version a bit confusing. What about keeping the same text as before, and just appending `*Since 0.56.0* can be a dictionary too.`? I think we append new behaviors in the doc in many other places like that.

Or reuse the phrasing you did below: `or, since 0.56.0, a dictionary of strings`